File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md

```diff
@@ -0,0 +1,160 @@
+
+
+Step 7: Select the target as **Azure** and click **Next** button
+![Select the target as Azure](Azure_Images/Functions_v1/Target_WordtoPDF.png)
+
+
+Step 8: Select the **Create new** button
+![Configure Hosting Plan](Azure_Images/Functions_v1/Function_Instance_WordtoPDF.png)
+
+
+Step 9: Click **Create** button 
+![Select the plan type](Azure_Images/Functions_v1/Subscription_detail_WordtoPDF.png)
+
+Step 10:After creating app service then click **Finish** button 
+![Creating app service](Azure_Images/Functions_v1/App_service_Created_WordtoPDF.png)
+
+
+Step 11: Click the **Publish** button
+![Click Publish Button](Azure_Images/Functions_v1/Before_Publish_WordtoPDF.png)
+
+Step 12: Publish has been succeed.
+![Publish succeeded](Azure_Images/Functions_v1/After_Publish_WordtoPDF.png)
+
+Step 13: Now, go to Azure portal and select the App Services. After running the service, click **Get function URL by copying it**. Then, paste it in the below client sample (which will request the Azure Functions, to perform **Word to PDF conversion** using the template Word document). You will get the output PDF document as follows.
+
+## Steps to post the request to Azure Functions:
+
+Step 1: Create a console application to request the Azure Functions API.
+
+Step 2: Add the following code snippet into **Main** method to post the request to Azure Functions with template Word document and get the resultant PDF document.
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Reads the template Word document.
+FileStream fs = new FileStream(@"../../Data/Input.docx", FileMode.Open, FileAccess.ReadWrite, FileShare.ReadWrite);
+fs.Position = 0;
+//Saves the Word document in memory stream.
+MemoryStream inputStream = new MemoryStream();
+fs.CopyTo(inputStream);
+inputStream.Position = 0;
+try
+{
+    Console.WriteLine("Please enter your Azure Functions URL :");
+    string functionURL = Console.ReadLine();
+    //Create HttpWebRequest with hosted azure functions URL.    
+    HttpWebRequest req = (HttpWebRequest)WebRequest.Create(functionURL);
+    //Set request method as POST
+    req.Method = "POST";
+    //Get the request stream to save the Word document stream
+    Stream stream = req.GetRequestStream();
+    //Write the Word document stream into request stream
+    stream.Write(inputStream.ToArray(), 0, inputStream.ToArray().Length);
+    //Gets the responce from the Azure Functions.
+    HttpWebResponse res = (HttpWebResponse)req.GetResponse();
+    //Saves the PDF document stream.
+    FileStream fileStream = File.Create("DocToPDF.pdf");
+    res.GetResponseStream().CopyTo(fileStream);
+    //Dispose the streams
+    inputStream.Dispose();
+    fileStream.Dispose();
+}
+catch (Exception ex)
+{
+    throw;
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).
```

Comment:
From GitHub, you can download the [console application](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Console_Application) and [Azure Functions v1](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).

Reply:
Updated